src/main/webapp/app/shared/participant-scores/participant-scores-average-table/participant-scores-average-table.component.ts

```diff
@@ -13,7 +13,7 @@ export class ParticipantScoresAverageTableComponent {
     @Input()
     isLoading = false;
 
-    extractParticipantName = (participantScoreAverageDTO: ParticipantScoreAverageDTO) => {
+    extractParticipantName = (participantScoreAverageDTO: any) => {
```

Comment:
Are you sure you want to use `any`?

Reply:
Tbh I'm not really sure in this place, the issue here is that I need the functions type to be of BaseEnitity -> string. But When I use the Base-entity type I get the issue that this type does not have the userName/teamName attributes. If you know a better way to do this, please tell me
![image](https://user-images.githubusercontent.com/33342534/115019856-0a5e7400-9eba-11eb-86fa-440b04eb9a3a.png)

Reply:
I managed to cast it inside the method, now the types are better